nodejs/Dockerfile-kali

```diff
@@ -0,0 +1,21 @@
+# A docker image for the func container.
+
+# debian variant is the official standard node image (larger than the alpine image)
+FROM kalilinux/kali-rolling
+
+RUN apt update && apt install -y curl nodejs
+RUN curl -L https://www.npmjs.com/install.sh | sh
```

Comment:
Similar to the above comment, probably sticking to a specific version would be nice!